account_invoice_base_invoicing_mode/models/sale_order.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    invoicing_mode = fields.Selection(
+        related="partner_invoice_id.invoicing_mode", readonly=True
```

Comment:
```suggestion
        related="partner_invoice_id.invoicing_mode"
```

Reply:
Well, I just noticed that it was first declared without the readonly, then it was added later on after a review.
I guess there is some good reason to have it or not !?

Reply:
readonly is true by default on v13 so you must specify it when you don't want it.

Reply:
Oh yes you are correct so it was unnecessary code :+1: